Src/Microsoft.Dynamic/zlib.net/Inflate.cs

```diff
@@ -487,7 +487,7 @@ internal int inflateSetDictionary(ZStream z, byte[] dictionary, int dictLength)
 		{
 			int index = 0;
 			int length = dictLength;
-			if (z == null || z.istate == null || z.istate.mode !=  InflateMode.DICT0)
+			if (z?.istate?.mode != InflateMode.DICT0)
```

Comment:
If z or z.istate are null then z?.istate?.mode is null so it's != InflateMode.DICT0. I agree it's not the most obvious conversion.